onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -396,10 +396,15 @@
     <string name="trip_list_context_showroute">Show route</string>
 
     <!--  Trip notifications -->
-    <string name="trip_stat_gone">Route %s has departed.</string>
-    <string name="trip_stat_lessthanone">Route %s is less than a minute away!</string>
-    <string name="trip_stat_one">Route %s is 1 minute away!</string>
-    <string name="trip_stat">Route %1$s is %2$d minutes away!</string>
+    <!--  Trip notifications -->
+    <string name="trip_stat_gone_arrived">Route %s has arrived.</string>
+    <string name="trip_stat_gone_departed">Route %s has departed.</string>
+    <string name="trip_stat_lessthanone_arriving">Route %s is arriving in less than a minute!</string>
```

Comment:
Let's change this and the following string to:

~~~
<string name="trip_stat_lessthanone_arriving">Route %s is arriving now!</string>
<string name="trip_stat_lessthanone_departing">Route %s is departing now!</string>
~~~

...so it matches with what we show in the arrivals list.